### ISSUE TITLE
It should report its own version and not the version of pylint

### ISSUE THREAD

**Issue by adiroiban** [twisted/twistedchecker#133]
If I run

```
$ twistedchecker --version
twistedchecker 1.5.6, 
astroid 1.4.9
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609]
```

I get the same version as pylint

```
$ pylint --version
No config file found, using default configuration
pylint 1.5.6, 
astroid 1.4.9
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609]
```

I think that it should report its own version, together with the versions for pylint :)

Or maybe re-wrote twistedchecker as a standard pylint plugin/checker
